airbyte-integrations/connectors/source-linkedin-ads/unit_tests/test_source.py

```diff
@@ -175,7 +175,7 @@ def test_next_page_token(self, requests_mock, response_json, expected):
         assert expected == result
 
     def test_request_params(self):
-        expected = "pageSize=500&q=search&search=(id:(values:List(1,2)))"
+        expected = "pageSize=500&q=search&search=(id:(values:List(urn%3Ali%3AsponsoredAccount%3A1,urn%3Ali%3AsponsoredAccount%3A2)))"
```

Comment:
should we add test config with account ids for check in cat to prevent similar issue in future?

Reply:
Added the new config